README.md

```diff
@@ -68,10 +63,13 @@ https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-d
 - [From Hashicorp](https://www.youtube.com/watch?v=YcJ9IeukJL8)
 - [Freecodecamp](https://www.youtube.com/watch?v=SLB_c_ayRMo)
 - [8 Best Pratices](https://www.youtube.com/watch?v=gxPykhPxRW0)
+- [Learn terraform ](https://youtu.be/iRaai1IBlB0)
```

Comment:
nitpick but I believe this should be 
```suggestion
- [Learn terraform](https://youtu.be/iRaai1IBlB0)
```
without the space.

Reply:
I like nitpicky! I want ALL the notes and suggestions! It's the only way to learn and grow.